### ISSUE TITLE
[REQUEST] Pause Sound when switching tabs

### ISSUE THREAD

**Issue by Satanarious** [ashpynov/PlayniteSound#11]
**Is your feature request related to a problem? Please describe.**
Currently, the sound/music for a game keeps playing even if you switch tabs and go to the home tab or switch between different tabs on the left panel.

**Describe the solution you'd like**
I would like an option to pause the sound playback when switched to another tab other than the library tab.

**Describe alternatives you've considered**
Right now, I have to pause the sound completely using the provided controls while switching from the library tab and turn them back on every time I switch to the library tab.

**Comment by ashpynov** [comment]
https://github.com/ashpynov/PlayniteSound/releases/tag/v5.7.16

**Comment by Satanarious** [comment]
Thank you so much; the new update fixed the problem.

**Comment by Satanarious** [comment]
This somehow created a bug, where switching between tabs will completely stop the playnite sound even if you come back to the library tab. The controls just show a pause button, which means that no soundtrack is selected somehow on returning to the games on the library tab. Switching between different games in the library tab also has no effect, you have to manually play a playnite sound by going into the game's setting->playnite sounds-><some music file>->play

**Comment by ashpynov** [comment]
Could you please share more info? which them are you using?

**Comment by Satanarious** [comment]
I tried the new version, and it fixed the bug I was facing. By the control button, I meant the one made available through different themes. Here, where there is a name for the track, only the pause button appeared when the bug existed and no track was selected. No bug was involved with the said payback/control button, but I indicated that to clarify things. And I think you understood what I was trying to say.
![image](https://github.com/user-attachments/assets/df1531e0-80f7-40cb-a1a8-22262e23ce32)

But with the latest update, this bug has been resolved. Thanks a lot.
